### ISSUE TITLE
UX | Profile Hub | Use recommendations from our research study to inform changes

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#69410]
## Background 
We completed the usability study and need to iterate on the hub as the cards were somewhat confusing for screen reader users. 

>[Recommendation 1](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/Research/2023-10-profile-hub-usability/research-findings.md#recommendations): Iterate on the Profile hub’s card content to make it easier for screen reader users to identify what is actionable/clickable.
Given that 1/2 screen reader participants encountered difficulty with the same words being used for both the card description and the link, they should be iterated upon so it’s more clear to screen reader users what is clickable.
See finding 7 for all supporting evidence

## Tasks 

- [x] Draft updated content
- [x] Review w. content specialist
- [x] Add any changes to Sketch files
- [x] Move to validate so PM knows it's ready for FE

## Acceptance Criteria 
- [x] we have a refreshed design that accounts for the recommendation made from the study 
- [x] Copies in sketch - Ready for FE

**Comment by mtcA6** [comment]
@pamela-drouin for next sprint I'd like for us to make updates to account for the recommendations from the study, do you mind making any updates to the tasks you think might be needed here to account for that work and giving this a size? 

cc: @Dottifacilities

**Comment by pamela-drouin** [comment]
[Mobile](https://www.sketch.com/s/b41a687f-8ab8-4be1-b189-692e5818cffd) and [Desktop](https://www.sketch.com/s/b41a687f-8ab8-4be1-b189-692e5818cffd) mocks have been updated!

**Comment by pamela-drouin** [comment]
Hmm, I don't remember discussing it, to be honest. @andaleliz?

**Comment by andaleliz** [comment]
No, I don't remember discussing it either. That's a good call out @mtcA6, I think we should add it. I know people use it, and it makes sense to be consistent with what we're doing in other sections. I can make that update.

**Comment by andaleliz** [comment]
@mtcA6 the [mobile](https://www.sketch.com/s/b41a687f-8ab8-4be1-b189-692e5818cffd/a/3ob2mwG) and [desktop](https://www.sketch.com/s/b41a687f-8ab8-4be1-b189-692e5818cffd/a/27Yp2GA) mock-ups have been updated with the DD view payment history link. This is ready for FE!

**Comment by mtcA6** [comment]
Links are added to the subsequent story
